Lib/_pydatetime.py

```diff
@@ -1473,6 +1474,11 @@ def tzinfo(self):
     def fold(self):
         return self._fold
 
+    def total_seconds(self):
```

Comment:
1. You should also update the C implementation.
2. You should add tests.
3. You should check corner cases where the arithmetic may be a fuzzy.